react/components/ProductSummaryAttachmentsList.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component, Fragment } from 'react'
+import { intlShape, injectIntl } from 'react-intl'
+import ProductPrice from 'vtex.store-components/ProductPrice'
+import { productShape } from '../propTypes'
+
+import { optionPricePerItem, parentPricePerUnit } from '../utils/attachmentHelper'
+
+class ProductSummaryAttachmentsList extends Component {
+  static propTypes = {
+    product: productShape,
+    intl: intlShape,
+  }
+
+  // TODO:  not sure if it is the best way to define if an item should be displayed
+  canShow = (option) => option.optionType !== 'Basic Toppings'
+
+  parentPrice = () => {
```

Comment:
`parentPriceByUnit` (?). And also, isn't there a look-alike function in the util?

Reply:
sim, esqueci de apagar essa ;)